english/invite-friends/smores/after_smores/indigestion.md

```diff
@@ -0,0 +1,7 @@
+Seating back after those satisfying smores, you suddenly get very tired. 
```

Comment:
It looks like there is no link to this new file.  Did you forget to commit that change?  Make sure you create (and commit) an option for each new "page" you create.